### PR TITLE
Feature: Burmese translation

### PR DESCRIPTION
**Pull request by enson0131**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution
Burmese locales were missing.

### 📝 Changelog
Added locales for Burmese translation.

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Added locales for Burmese translation.        |
| 🇨🇳 Chinese |   添加缅甸语        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
Should update the snapshot for test case.

File: docs/react/i18n.en-US.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ The following languages are currently supported:
 | Chinese (Simplified)     | zh_CN    |
 | Chinese (Traditional)    | zh_HK    |
 | Chinese (Traditional)    | zh_TW    |
+| Myanmar                  | my_MM    |
```
Comment (yoyo837) [inline docs/react/i18n.en-US.md:100]: `Burmese` ?

Comment (enson0131) [reply docs/react/i18n.en-US.md:100]: OK。I modify it

**Review by yoyo837 — APPROVED** [review]

**Comment by yoyo837** [comment]
已发布

**Pull request merged**